experiments/ssl4eo/sample_conus.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import csv
+import os
+import random
+
+import fiona
+from shapely.geometry import Point, shape
+from torchvision.datasets.utils import download_and_extract_archive
+
+
+def get_uniform_points_within_conus(
+    download_root: str, num_samples: int
+) -> list[tuple[float, ...]]:
+    random.seed(0)
```

Comment:
I like this. We should probably add this to the other sampling script too. Either in this PR or a separate PR.

Reply:
See https://github.com/microsoft/torchgeo/pull/1318#issuecomment-1540891280, might solve the issue here.